### PR TITLE
fix: Vue sdk RELEASE

### PR DESCRIPTION
**Pull request by nirgur**
## Related Issues

Related https://github.com/descope/etc/issues/6416

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": [
-    "local>descope/renovate-config"
-  ]
+  "extends": ["local>descope/renovate-config"],
+  "enabled": false
```
Comment (Bars92) [inline renovate.json:4]: renovate disabled completely here ☝️ need to revisit later

File: packages/widgets/user-management-widget/e2e/user-management-widget.spec.ts [hunk 2]
```diff
@@ -678,13 +682,10 @@ test.describe('widget', () => {
...
     await searchInput.fill('mockSearchString');
 
-    // wait for results to filter
-    await page.waitForTimeout(5000);
-
     // only search results shown in grid
     await expect(
       page.locator(`text=${mockUsers[0]['loginIds'][0]}`).first(),
-    ).toBeHidden();
+    ).toBeHidden({ timeout: 20000 });
```
Comment (Bars92) [inline packages/widgets/user-management-widget/e2e/user-management-widget.spec.ts:688]: kinda overkill

**Review by Bars92 — COMMENTED** [review]
@nirgur not sure about vue but the rest is good. Lets merge this asap @tomerlichtash @asafshen

File: renovate.json (hunk 1 above)
Comment (orius123) [reply renovate.json:4]: @omercnet we will still get security issues from Dependabot, right?

File: packages/widgets/user-management-widget/e2e/user-management-widget.spec.ts (hunk 2 above)
Comment (orius123) [reply packages/widgets/user-management-widget/e2e/user-management-widget.spec.ts:688]: true

Comment (orius123) [reply packages/widgets/user-management-widget/e2e/user-management-widget.spec.ts:688]: ci is slow

**Review by orius123 — APPROVED** [review]

**Pull request merged**